scripts/biomedical/chembl/test-data/chembl_uniprot_expected_data.csv

```diff
@@ -0,0 +1,17 @@
+uniprotID,ChemBL,Name,Protein_Type,dcid
```

Comment:
again replace _ with spaces in the name. again text values aren't formatted properly